lib/rubocop/cop/style/braces_around_hash_parameters.rb

```diff
@@ -109,11 +109,14 @@ def check_context_dependent(arg, args)
         # and that block uses do..end. The reason for wanting to check this is
         # that the do..end could bind to a different method invocation if the
         # hash braces were removed.
+        #
+        # Another case for true return value is when you splat extra keyword
+        # arguments into the last argument like: `f({val: 1, **extra})`.
         def braces_needed_for_semantics?(arg)
           arg.each_pair do |_key, value|
```

Comment:
`#each_pair` does not iterate over the splat, that's why I re-iterate looking for it with the `#children` call below.